### PR TITLE
Cart

### PR DESCRIPTION
**Pull request by EntropicWind**
added pinpoint icon and made changes to numeric value

### CODE REVIEW

File: components/cart.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+}
+
+    return (
+    <div className={`${style.container}`}> 
+        <div>
+            <div className={`${style.pricing}`}>
+                <div className="numbers">
+                    <span id="symbol">$</span>
+                    <span id="dollars">17</span>
+                    <span id="decimal">.</span>
```
Comment (moses369) [inline components/cart.js]: Possible unnecessary spans, why have each letter on its own span?

**Review by moses369 — COMMENTED** [review]
Considering we're using nextjs did you mean to leave the ids, that don't have any styling attached to them?

File: components/cart.js (hunk 1 above)
Comment (EntropicWind) [reply components/cart.js]: was going with the possibility of dynamic adjusting to the price

Comment (EntropicWind) [reply components/cart.js]: oh no thats the old ones, I changed those in the latest push, thats how I got the cents to be small and to the top

**Review by JacobDWalters — APPROVED** [review]

**Review by moses369 — APPROVED** [review]

**Pull request merged**